homework-submission/my app/my-app/src/add.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+class AddTodo extends React.Component {
+  state = {
+    description: '',
+    done: false
+  }
+  todo = (element) => {
+    this.setState({ [element.target.id]: element.target.value })
+  }
+  submit = (element) => {
+    element.preventDefault();
+    this.props.addTodo(this.state)
+    this.setState({
+      description: '',
+      done: false,
+    })
+  }
+  render() {
+    return (
+      <div>
+        <form onSubmit={this.submit}>
+          <span>Add Todo</span>
+          <div>
+            <span>Todo description :</span>
+            <input type='text' id='description' onChange={this.todo}>
+            </input>
```

Comment:
you may have a self close tag here instead of a complete element
```HTML
<h1></h1> <!-- this is an element, it can hold content -->
<img /> <!-- this is a tag, it can't holds any content inside of it. -->
```